.github/workflows/server-ci-master.yml

```diff
@@ -0,0 +1,14 @@
+name: Server CI
```

Comment:
I think we should use a unique naming on the workflows as it quickly become conusing on the Ations UI:
<img width="312" alt="resim" src="https://user-images.githubusercontent.com/2153367/236991184-db4a4dc3-dd9c-4a10-a134-bdb0657e46a9.png">